airbyte-integrations/connectors/source-amazon-ads/acceptance-test-config.yml

```diff
@@ -5,15 +5,14 @@ tests:
   spec:
     - spec_path: "integration_tests/spec.json"
   connection:
-    - config_path: "secrets/config.json"
+    - config_path: "secrets/config_report.json"
```

Comment:
Can you please explain what the purpose of this change is?

This connector has two config files for different tests. They are configured in CI, and this change might introduce a regression.

I suggest continuing to keep both configs as they were.

Reply:
added back in